ToDo.DataAccess/ToDo.DataAccess/DataBase/ToDoDbContext.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.EntityFrameworkCore;
```

Comment:
Remove unused usings everywhere.

Reply:
this namespace is needed for db, others unused namespaces were removed